Source/Core/Core/HW/DSP.cpp

```diff
@@ -145,7 +145,7 @@ struct ARAMInfo
 
 // STATE_TO_SAVE
 static ARAMInfo g_ARAM;
-static UDSPControl g_dspState;
+UDSPControl g_dspState;
```

Comment:
This should be moved outside of the static values for consistency (being put at the bottom of the list, or the top)